### ISSUE TITLE
More ways to add people to Direct Messages chat

### ISSUE THREAD

**Issue by zdumitru** [RocketChat/feature-requests#323]
**Is your feature request related to a problem? Please describe.**
With the recent introduction of direct messages between multiple users, it is only possible to create new direct messages chats from the small button in the top left corner.

I suggest an extra way of creating direct messages chats between multiple parties:
- Choose one participant (clicking on desired username) and then adding more people from the middle screen (new button) or right panel (new button).

**Describe the solution you'd like**
Add a new button to the direct message screen alowing to add more participants

**Describe alternatives you've considered**
Make `Direct Messages` text in the left panel clickable. That would allow to choose participants.

**Additional context**

![image](https://user-images.githubusercontent.com/11751017/77771416-f04fba80-7046-11ea-8a5a-e8683f560d92.png)

**Comment by ggazzo** [comment]
do you agree, adding a new user should create a new room?

**Comment by zdumitru** [comment]
@ggazzo Yes, new user combinations should create a new DM room with these users.
Users combination that had past conversations, should switch to that room.

**Comment by 4nu81** [comment]
@zdumitru @ggazzo I disagree. I don't see any reason not to add another person to a group-direct-chat with making him able to see all prior messages.

**Comment by s2k** [comment]
It should be possible to add a new user to a DM room, _especially_ in a work environment. We often meet using DMs, and forgetting to add a user in the beginning or needing to add one later can happen.
In these cases, it's annoying to have to create a new DM room since, first of all, it requires extra work and second of all, the newly added members lack the context that we previously talked about.

**Comment by smooti** [comment]
Is there any update on this issue. This feature would be very convenient as recreating an entire room because you want another user to be aware of a group conversation is impractical.

**Comment by wpost** [comment]
Need a user story before you are convinced this would be useful? Here you go:

As a team manager, I want to be able to add a new hire to a group DM so that the new hire is easily onboarded and can see previous discussions.

Not a hypothetical example. We're facing that at work today. Yes, in hindsight our team should have been communicating in a channel, not a group DM. But surely we're not the only ones who would find the requested feature useful.

**Comment by ASponch** [comment]
Hi,

If the DM owner feels it is necessary to invite another member, this seems like an interesting option.

Best regards

**Comment by PleasantD** [comment]
Adding and removing people from a direct group chat would be a huge benefit.
Especially if the new users are able to see the original history of the chat.

Additionally, being able to name the chat would be a benefit.
This allows for small teams to form dynamically around a single shared discussion.
Quite often this is topic centric and new people may be needed as the discussion evolves, or members may want to leave when they are no longer needed in the conversation.

At some point the conversation topic ends and the temporary "room" is no longer needed, but may be kept around for archival purposes. 

Instead of "removing" a user, being able to "mute" a direct group chat would also be fine, as this would preserve history.

An example of this would be Facebook Messenger chats.
